### PR TITLE
[doc]Download and ingest

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Add download-and-ingest

### CODE REVIEW

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+|        | ---- edge-${FIRST_KEY_IN_THIS_FILE}.sst
+|---2
+         | ---- vertex-${FIRST_KEY_IN_THIS_FILE}.sst
+         | ---- edge-${FIRST_KEY_IN_THIS_FILE}.sst
+....
+```
+
+其中各个目录为 partition 编号。<br />SST 文件名格式为 `{TYPE}-${FIRST_KEY_IN_THIS_FILE}.sst`，其中 `TYPE` 为 `vertex` 或 `edge`，`FIRST_KEY_IN_THIS_FILE` 为文件中的起始 key。（如果你想自己写工具生成 sst 文件，需要保证每个 `SST`  文件中的 key 是有序的。)
+
+请确认所有 server 已安装 `Hadoop`，并且 `HADOOP_HOME ` 已设置。<br />运行 nebula console，执行 Download 命令：
```
Comment (whitewum) [inline docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: why <br /> but not just enter?

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Download and Ingest
+
+Nebula 存储访问默认使用 `RocksDB` 作为 `key-value` 存储引擎。因此在大量数据加载时，可以通过运行一个 map-reduce job 离线生成 RocksDB 的 sst 文件，再直接分发到服务器上。
+
+Nebula 提供了 `Spark-SSTFile-Generator` 工具。
+
+`Spark-SSTFile-Generator` 通过映射文件，从 hive 表生成 SST 文件。<br />具体用法详见 [Spark application command line reference](https://github.com/vesoft-inc/nebula/blob/master/src/tools/spark-sstfile-generator/README.md)。
```
Comment (whitewum) [inline docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: why not just enter ?

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Download and Ingest
+
+Nebula uses `RocksDB` as the default `key-value` storage engine. Therefore, when a large amount of data is loaded, the sst files of RocksDB can be generated offline by running a map-reduce job and distributed directly to the servers.
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: when you are trying to load a large amount of data,

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Nebula provides `Spark-SSTFile-Generator` tool.
+
+`Spark-SSTFile-Generator` generates SST files from the hive table via the mapping files. For details on how to use it, please refer [Spark application command line reference] (https://github.com/vesoft-inc/nebula/blob/master/src/tools/spark-sstfile-generator/README.md).
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: preview to see errors

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Download the `SST` files of each server into directory `data/download` respectively via the `DOWNLOAD` command and `meta` in the storage servers. Explanation of the above command:
+
+- HADOOP_HOST specifies Hadoop NameNode address
+- HADOOP_PORT specifies Hadoop NameNode port number
+- HADOOP_PATH specifies Hadoop data storage directory
+
+If error occurs when downloading, delete the corresponding data files in `data/download` directory and try to download again. If error occurs again, please raise us an issue on [GitHub](https://github.com/vesoft-inc/nebula/issues). <br /> When data download is done, re-execute the command leads to no actions.
+
+When the offline sst data download is done, it can be loaded into the storage service via `INGEST` command.
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: ingest into.
load is different to ingest

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+`INGEST` command is as follows:
+
+```bash
+nebula > INGEST
+```
+
+The command will load the `SST` files in `data/download` directory.
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: ingest

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+# Download and Ingest
+
+Nebula 存储访问默认使用 `RocksDB` 作为 `key-value` 存储引擎。因此在大量数据加载时，可以通过运行一个 map-reduce job 离线生成 RocksDB 的 sst 文件，再直接分发到服务器上。
```
Comment (darionyaphet) [inline docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: SST

File: docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+|        | ---- edge-${FIRST_KEY_IN_THIS_FILE}.sst
+|---2
+         | ---- vertex-${FIRST_KEY_IN_THIS_FILE}.sst
+         | ---- edge-${FIRST_KEY_IN_THIS_FILE}.sst
+....
+```
+
+其中各个目录为 partition 编号。
+
+SST 文件名格式为 `{TYPE}-${FIRST_KEY_IN_THIS_FILE}.sst`，其中 `TYPE` 为 `vertex` 或 `edge`，`FIRST_KEY_IN_THIS_FILE` 为文件中的起始 key。（如果你想自己写工具生成 sst 文件，需要保证每个 `SST`  文件中的 key 是有序的。)
```
Comment (darionyaphet) [inline docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: 其中 `TYPE` 表示数据类型

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+# Download and Ingest
+
+Nebula uses `RocksDB` as the default `key-value` storage engine. Therefore, when you are trying to load a large amount of data, the sst files of RocksDB can be generated offline by running a map-reduce job and distributed directly to the servers.
```
Comment (darionyaphet) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: ditto

File: docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+Download the `SST` files of each server into directory `data/download` respectively via the `DOWNLOAD` command and `meta` in the storage servers. Explanation of the above command:
+
+- HADOOP_HOST specifies Hadoop NameNode address
+- HADOOP_PORT specifies Hadoop NameNode port number
+- HADOOP_PATH specifies Hadoop data storage directory
+
+If error occurs when downloading, delete the corresponding data files in `data/download` directory and try to download again. If error occurs again, please raise us an issue on [GitHub](https://github.com/vesoft-inc/nebula/issues). When data download is done, re-execute the command leads to no actions.
+
+When the offline sst data download is done, it can be ingested into the storage service via `INGEST` command.
```
Comment (darionyaphet) [inline docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-and-ingest-syntax.md]: ditto

**Comment by darionyaphet** [comment]
jenkins go

**Pull request merged**